### PR TITLE
integ: activate YAML extension in necessary test

### PR DESCRIPTION
**Pull request by JadenSimon**
## Problem
Red hat broke up the YAML extension into multiple components, breaking any YAML integration tests.

## Solution
Activate the YAML extension in the necessary test

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/shared/extensions.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export const VSCODE_EXTENSION_ID = {
     // python depends on jupyter plugin
     jupyter: 'ms-toolsai.jupyter',
     yaml: 'redhat.vscode-yaml',
+    redhatCommon: 'redhat.vscode-commons',
```
Comment (justinmk3) [inline src/shared/extensions.ts:20]:
```suggestion
    // For redhat yaml extension.
    redhatCommon: 'redhat.vscode-commons',
```

**Review by justinmk3 — APPROVED** [review]

**Comment by JadenSimon** [comment]
I guess you can't install the extension on 1.42, and it looks like we need to force activate the YAML extension before starting tests.

**Comment by justinmk3** [comment]
> I guess you can't install the extension on 1.42, and it looks like we need to force activate the YAML extension before starting tests.

Then we should just disable the relevant tests for 1.42.

File: src/integrationTest/shared/codelens/samTemplateCodeLensProvider.test.ts [hunk 2]
```diff
@@ -32,6 +34,8 @@ describe('SamTemplateCodeLensProvider', async function () {
             this.skip()
         }
 
+        await activateExtension(VSCODE_EXTENSION_ID.yaml)
```
Comment (justinmk3) [inline src/integrationTest/shared/codelens/samTemplateCodeLensProvider.test.ts:37]:
```suggestion
        // Note: redhat.vscode-yaml no longer works on vscode 1.42
        await activateExtension(VSCODE_EXTENSION_ID.yaml)
```

**Comment by JadenSimon** [comment]
Integration tests all passed, merging this.

**Pull request merged**